ui/v2.5/src/docs/en/ScrapeOperation.md

```diff
@@ -0,0 +1,49 @@
+# Metadata Scraping
+
+Stash supports scraping of metadata from variuos external sources
+
+# Termonology
+  ### URL Scraper
+  * A URL scraper is a sctaper that attempts to extract metadata from a given URL
+  * high chance of scuccess
+  ### Fragment Scraper 
+  * This type of scraper will attempt to use all current metadata for an Item and match it to a result from a metadata source automatically
+  * lower chance of success
+  ### Search/By Name Search
+  * This type of scraper will use the current name of the Item to search a given metadata source for a list of matches
+  * This scraper relies on the user picking the correct match from a list of results 
+
+# Supported Scrapers:
+
+|   | URL | Search | Fragment |
+|---|:---:|:---:|:---:|
+| gallery | :heavy_check_mark: | | :heavy_check_mark: |
+| movie | :heavy_check_mark: | | |
+| performer | :heavy_check_mark: | :heavy_check_mark: |   |
+| scene | :heavy_check_mark: | :heavy_check_mark:| :heavy_check_mark: |
+
+# Scraper Operation
+
+Stash has a built in performer `search` scraper for freeones.xxx.
```

Comment:
Seems odd having this here on its own. Probably needs a sub-heading or something.